libr/include/r_io.h

```diff
@@ -458,6 +460,8 @@ R_API bool r_io_fd_get_base (RIO *io, int fd, ut64 *base);
 R_API const char *r_io_fd_get_name (RIO *io, int fd);
 R_API int r_io_fd_get_current(RIO *io);
 R_API bool r_io_use_fd (RIO *io, int fd);
+R_API int r_io_fd_get_next (RIO *io, int fd);
```

Comment:
fds are int or ut32? :D

Reply:
int, they are int, and ids are ut32. There is no point in having int ids, but it's useful to have int fds